x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats_count_distinct.csv-spec

```diff
@@ -177,3 +177,70 @@ m:long  | languages:i
 20   | 5
 10   | null
 ;
+
+foldablePrecision
+required_capability: extend_aggs_on_constants_support
+from employees | eval x = 82+8 | stats count_distinct(salary, x*100);
+
+count_distinct(salary, x*100):long
+100
```

Comment:
I think the data set is too small to test that the precision is actually correctly propagated. Shouldn't we add a test on a dataset where different values for the precision lead to different `count_distinct` results?